src/api_server/src/lib.rs

```diff
@@ -314,12 +314,14 @@ mod tests {
     use utils::tempfile::TempFile;
     use utils::time::ClockType;
     use vmm::builder::StartMicrovmError;
+    use vmm::cpu_config::templates::test_utils::TEST_UNESCAPED_JSON_TEMPLATE;
```

Comment:
nit: how about having the test input in `src/api_server/src/lib.rs` with `#[cfg(target_arch="<arch>")]`, since they are only used in the file?

Reply:
I need to rebase and merge commits in any event. I can add this. Although those methods are only used in this file for the moment, I have generally preferred to avoid using `target_arch` annotations, but this is my personal preference only. Another option would be to add the x86_64 template as a hard-coded string into the lib.rs file and then only run the test on x86_64. wdyt?

Reply:
> Another option would be to add the x86_64 template as a hard-coded string into the lib.rs file and then only run the test on x86_64. wdyt?

This option also requires the `target_arch` annotation to run the test only on x86.

> Although those methods are only used in this file for the moment, I have generally preferred to avoid using target_arch annotations, but this is my personal preference only.

I also prefer to avoid using arch-dependent code, but having the test cases in different files looks worse for me. It's a matter of taste though.